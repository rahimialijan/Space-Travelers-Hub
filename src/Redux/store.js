import { configureStore } from '@reduxjs/toolkit';
import missionsReducer from './mission/missionSlice';

const store = configureStore({
  reducer: {
    missions: missionsReducer,
  },
});

export default store;
