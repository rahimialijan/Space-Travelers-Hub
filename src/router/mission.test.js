import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../Redux/store';

import Missions from './mission';

test('renders Missions component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Missions />
    </Provider>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
