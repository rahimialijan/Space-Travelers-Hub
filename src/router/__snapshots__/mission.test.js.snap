// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Missions component correctly 1`] = `
<div>
  <table
    className="striped-table"
  >
    <thead>
      <tr>
        <th
          className="nowrap"
        >
          Mission Name
        </th>
        <th
          className="nowrap"
        >
          Description
        </th>
        <th
          className="nowrap"
        >
          Status
        </th>
        <th
          className="nowrap"
        >
          Actions
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
